src/layouts/navbar/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { NavbarItems } from "./NavbarItems";
+import { FaSearch } from "react-icons/fa";
+import "./style.css";
+const Navbar = () => {
+  return (
+    <>
+      <nav className="navbar">
+        <ul>
+          {NavbarItems.map((item, index) => (
+            <li key={index}>
+              <a href={`/${item.toLowerCase()}.html`}>{item}</a>
```

Comment:
instead of writing lower case , write css property text tranform uppercase, and update navbar arr all in small